rdr_service/offline/response_validation.py

```diff
@@ -95,6 +112,23 @@ def _send_error_message(self):
         result_text += '\n'.join(sorted(result_list))
         self._output_result(result_text)
 
+    def _insert_data(self):
+        for key in self._error_list.items():
+            survey_code, question_code, error_str, error_type, pid, answer_id = key
+
+            data = PpiValidationErrors(
+                eval_date=self._since_date,
```

Comment:
Should this be set to the current datetime, rather than the start of the time window being evaluated?

Reply:
Oh yeah, that makes more sense. I will update it now